libs/gretty/src/main/groovy/org/akhikhl/gretty/ServerConfigWithInputs.java

```diff
@@ -0,0 +1,172 @@
+package org.akhikhl.gretty;
+
+import groovy.lang.Closure;
+import org.gradle.api.model.ReplacedBy;
+import org.gradle.api.tasks.Console;
+import org.gradle.api.tasks.Input;
+import org.gradle.api.tasks.InputFile;
+import org.gradle.api.tasks.Internal;
+import org.gradle.api.tasks.Optional;
+import org.gradle.api.tasks.PathSensitive;
+import org.gradle.api.tasks.PathSensitivity;
+
+import java.util.List;
+import java.util.Map;
+
+public interface ServerConfigWithInputs {
```

Comment:
Please don't mix. Either

1. place .java source files in src/main/java, or
2. make this file .groovy and apply some groovyfication (stripping `;`, for example)

I would suggest (2), it's a mostly Groovy project, and a groovy-only module.

Reply:
I did 2)